doc/how-to/replication/repl_bootstrap_auto.rst

```diff
@@ -0,0 +1,437 @@
+Set :ref:`iproto.advertise.peer <configuration_reference_iproto_advertise_peer>` to advertise the current instance to other replica set members:
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/auto_leader/config.yaml
+    :language: yaml
+    :start-at: iproto:
+    :end-at: peer: replicator@
+    :dedent:
+
+.. _replication-automated-failover_configuring_result:
+
+Resulting configuration
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The resulting replica set configuration should look as follows:
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/auto_leader/config.yaml
+    :language: yaml
+    :end-at: listen: 127.0.0.1:3303
+    :dedent:
+
+
+
+
+.. _replication-automated-failover-work:
+
+Working with a replica set
+--------------------------
+
+.. _replication-automated-failover-start-instances:
+
+Starting instances
+~~~~~~~~~~~~~~~~~~
+
+1.  After configuring a replica set, execute the :ref:`tt start <tt-start>` command from the :ref:`tt environment directory <replication-automated-failover-tt-env>`:
+
+    .. code-block:: console
+
+        $ tt start auto_leader
+           • Starting an instance [auto_leader:instance001]...
+           • Starting an instance [auto_leader:instance002]...
+           • Starting an instance [auto_leader:instance003]...
+
+2.  Check that instances are in the ``RUNNING`` status using the :ref:`tt status <tt-status>` command:
+
+    .. code-block:: console
+
+        $ tt status auto_leader
+        INSTANCE                    STATUS      PID
+        auto_leader:instance001     RUNNING     24768
+        auto_leader:instance002     RUNNING     24769
+        auto_leader:instance003     RUNNING     24767
+
+
+
+.. _replication-automated-failover-work-status:
+
+Checking a replica set status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1.  Connect to ``instance001`` using :ref:`tt connect <tt-connect>`:
+
+    .. code-block:: console
+
+        $ tt connect auto_leader:instance001
+           • Connecting to the instance...
+           • Connected to auto_leader:instance001
+
+2.  Check the instance state in regard to :ref:`leader election <repl_leader_elect_process>` using :ref:`box.info.election <box_info_election>`.
+    The output below shows that ``instance001`` is a follower while ``instance002`` is a replica set leader.
+
+    .. code-block:: console
+
+        auto_leader:instance001> box.info.election
+        ---
+        - leader_idle: 0.77491499999815
+          leader_name: instance002
+          state: follower
+          vote: 0
+          term: 2
+          leader: 1
+        ...
+
+3.  Check that ``instance001`` is in read-only mode using ``box.info.ro``:
+
+    .. code-block:: console
+
+        auto_leader:instance001> box.info.ro
+        ---
+        - true
+        ...
+
+4.  Execute ``box.info.replication`` to check a replica set status.
+    Make sure that ``upstream.status`` and ``downstream.status`` are ``follow`` for ``instance002`` and ``instance003``.
```

Comment:
```suggestion
    Make sure that ``upstream.status`` and ``downstream.status`` are ``follow`` for ``instance001`` and ``instance003``.
```

Reply:
> Make sure that ``upstream.status`` and ``downstream.status`` are ``follow`` for ``instance002`` and ``instance003``.

Looks like the description is correct:

<img width="849" alt="image" src="https://github.com/tarantool/doc/assets/38073144/9a266e74-4123-46f1-b495-367b77caadb1">